src/main/java/uk/gov/justice/digital/delius/controller/secure/UserController.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import io.swagger.annotations.*;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.justice.digital.delius.controller.advice.ErrorResponse;
+import uk.gov.justice.digital.delius.data.api.StaffDetails;
+import uk.gov.justice.digital.delius.data.api.UserDetails;
+import uk.gov.justice.digital.delius.service.UserService;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+@RestController
+@RequestMapping(value = "secure", produces = MediaType.APPLICATION_JSON_VALUE)
+@Api(tags = "User API", authorizations = {@Authorization("ROLE_COMMUNITY")})
+@AllArgsConstructor
+@PreAuthorize("hasRole('ROLE_COMMUNITY')")
```

Comment:
I think I would be tempted to protect this with the `ROLE_AUTH_DELIUS_LDAP` role instead. There can be argument that this returns sensitive information about our uses and would be consistent with the AuthenticationController that returns similar data. Obviously the pathfinder client will need to have that role added, but that is trivial

Reply:
agreed and changed